RELEASE-MASTERS.md

```diff
@@ -1,10 +1,85 @@
-# Releasing Knative
+# Release Masters
+
+For each release cycle, we dedicate a team of two individuals, one from Eventing
+and one from Serving, to shepherd the release process. Participation is
+voluntary and based on good faith. We are only expected to participate during
+our local office hour.
+
+# Roster
+
+We seed this rotation with all approvers from all the Serving and Eventing
+workgroups, excluding productivity. If you are no longer active in Knative, or
+if you are contributing on personal capacity and do not have time to contribute
+in the rotation, feel free to send a PR to remove yourself.
+
+## Serving roster
+- dprotaso
+- julz
+- JRBANCEL
+- markusthoemmes
+- mattmoor
+- nak3
+- tcnghia
+- vagababov
+- yanweiguo
+- ZhiminXiang
```

Comment:
@mdemirhan seems a networking WG lead? :)

Reply:
Observability approver :)

Reply:
Observability WG is being archived, and I think we will soon make some changes to the networking WG personel list